openedx-unsupported/configuration
# [SE-3111] Update labxchange-xblocks audio xBlock

See https://github.com/open-craft/labxchange-xblocks/pull/11 - this pull request adds transcripts support for audio xBlock.

**Dependencies**: None
**Merge deadline**: None
**Testing instructions**:

- Verify that the updated hash/revision of the labxchange-xblocks dependency is valid and matches the commit that merged open-craft/labxchange-xblocks#11
- Provision the platform with this configuration
- Verify that labxchange-xblocks is installed successfully in the environment.

**Reviewers**

 - @s0b0lev
 - edX reviewer[s] TBD

**Settings**
```
```
